NoSQL/ce-sec/Dockerfile

```diff
@@ -20,6 +20,18 @@ ENV KV_SERVICERANGE 5021-5049
 # specific to secure configuration
 RUN microdnf -y install openssl-1.1.1k && microdnf clean all
 
+# hadolint ignore=DL4006
```

Comment:
i think the user name should be something different - something other than product name.